snyk-monitor/README.md

```diff
@@ -308,6 +308,17 @@ helm upgrade --install snyk-monitor snyk-charts/snyk-monitor \
   --set volumes.projected.serviceAccountToken=true
 ```
 
+## Using AKS with Managed Identities
+
+If you are using AKS with Managed Identities to authorize access to ACR, you must also specify the Client ID of the user-managed identity that assigns the `AcrPull` role to the VM scale set of that cluster. This value must be set as an override, in `.Values.azureEnvVars`:
```

Comment:
If the user uses a `system-assigned` identity, then they won't need this step, and possibly just 1 `user-assigned` identity (not sure about this one 🤔). 
This env var would only need to be populated when the set up of the Managed Identity has multiple user-assigned managed identities. Maybe we should be explicit here to avoid further questions from users 😅